src/Calculations.ts

```diff
@@ -4,5 +4,46 @@ import { Point3D } from './DataTypes';
 /* Compute the angle between the first three Point3Ds in the provided Array
  * in degrees, with index 0 as theta. */
 export function computeAngle(points: Array<Point3D|null>): number {
-    return 178.600363111;
+
+    if (points.length < 3)
+        return NaN;
```

Comment:
I think this would be better as null, since NaN is annoying to check and TS won't enforce it